src/packages/excalidraw/README.md

```diff
@@ -393,6 +393,7 @@ To view the full example visit :point_down:
 | [`libraryReturnUrl`](#libraryReturnUrl) | string |  | What URL should [libraries.excalidraw.com](https://libraries.excalidraw.com) be installed to |
 | [`theme`](#theme) | `light` or `dark` |  | The theme of the Excalidraw component |
 | [`name`](#name) | string |  | Name of the drawing |
+| [`UIOptions`](#UIOptions) | <pre>{ <br> canvasActions?: { <br> changeViewBackgroundColor?: boolean,<br> clearCanvas?: boolean,<br> export?: boolean,<br> loadScene?: boolean,<br> saveAsScene?: boolean,<br> saveScene?: boolean,<br> theme?: boolean <br> }<br>}</pre> |  | To hide certain UI elements |
```

Comment:
We have `README_NEXT.md` for next version, can you revert this readme move the readme changes to `README_NEXT` ?